spikingjelly/activation_based/triton_kernel/flex_sn_inductor/hop.py

```
@@ -135,6 +221,560 @@ def eager_scan(
+    result = tuple(result)
+    return result[num_states:]
+
+
+def lowerable_scan_final_state(
+    core_fn: Callable,
+    num_inputs: int,
+    num_states: int,
+    num_outputs: int,
+    *flat_args: torch.Tensor,
+) -> Tuple[torch.Tensor, ...]:
+    if _torch_scan_op is None or _wrap_scan_combine_fn_flat is None:
```

Ruff (0.15.11)

[error] Undefined name `_wrap_scan_combine_fn_flat`

(F821)